portal/src/main/webapp/js/src/cbio-util.js

```diff
@@ -552,9 +552,26 @@ cbio.util = (function() {
             maxValue = Math.ceil(q3 + iqr * 1.5);
             minValue = Math.floor(q1 - iqr * 1.5);
         }
-        if(maxValue > values[values.length - 1])maxValue = values[values.length - 1];
         if(minValue < values[0])minValue = values[0];
-        // Then return
+        if(maxValue > values[values.length - 1])maxValue = values[values.length - 1];
+        //provide the option to choose min and max values from the input array
```

Comment:
Maybe include this explanation above the function. 
I would recommend jsdoc style
/**
 * ....
 * @param {Array} data 
 * @param {Boolean} inArrayFlag
 * @return {Array} outliers
 */